web/profiles/custom/os2loop/modules/os2loop_flag_content/src/Form/FlagContentAdminForm.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\os2loop_flag_content\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Configure flag content admin settings for this site.
+ */
+class FlagContentAdminForm extends ConfigFormBase {
+
+  /**
+   * Config settings.
+   *
+   * @var string
+   */
+  const SETTINGS = 'os2loop_flag_content.settings';
```

Comment:
`os2loop_flag_content.settings` should be added to the Config ignore settings (/admin/config/development/configuration/ignore) to make sure that it's not overwritten on config import.